install_ffmpeg.sh

```diff
@@ -33,7 +33,7 @@ if [ ! -e "$HOME/ffmpeg/libavcodec/libavcodec.a" ]; then
   git clone https://github.com/livepeer/FFmpeg.git "$HOME/ffmpeg" || echo "FFmpeg dir already exists"
   cd "$HOME/ffmpeg"
 
-  git checkout 16b8686687413ef99fa398ca63585ff438737378
+  git checkout 86997f6261671327ee05d598f38acabf8ebfd088
```

Comment:
`chmod -x` needed on this file

Reply:
Alternatively we should add `#!/usr/bin/env bash` at the top of the file, to make it executable like `./install_ffmpeg.sh` similar to what's done in `go-livepeer`